template.tpl

```diff
@@ -502,7 +502,10 @@ const mergeObj = (obj, obj2) => {
 const parseEecObj = prod => {
   return {
     id: prod.id,
-    quantity: prod.quantity
+    quantity: (prod.quantity || 1),
```

Comment:
added a fallback of 1 for quantity for each item in contents array (quantity should always be at least 1...)